src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowCreate.cs

```diff
@@ -74,50 +88,13 @@ public override DbCommand GetCommand(DbConnection connection)
         {
             Validate.IsNotNull(nameof(connection), connection);
 
-            // Process all the columns. Add the column to the output columns, add updateable
-            // columns to the input parameters
-            List<string> outColumns = new List<string>();
-            List<string> inColumns = new List<string>();
-            DbCommand command = connection.CreateCommand();
-            for (int i = 0; i < AssociatedResultSet.Columns.Length; i++)
-            {
-                DbColumnWrapper column = AssociatedResultSet.Columns[i];
-                CellUpdate cell = newCells[i];
-
-                // Add the column to the output
-                outColumns.Add($"inserted.{SqlScriptFormatter.FormatIdentifier(column.ColumnName)}");
-
-                // Skip columns that cannot be updated
-                if (!column.IsUpdatable)
-                {
-                    continue;
-                }
-
-                // If we're missing a cell, then we cannot continue
-                if (cell == null)
-                {
-                    throw new InvalidOperationException(SR.EditDataCreateScriptMissingValue);
-                }
-
-                // Create a parameter for the value and add it to the command
-                // Add the parameterization to the list and add it to the command
-                string paramName = $"@Value{RowId}{i}";
-                inColumns.Add(paramName);
-                SqlParameter param = new SqlParameter(paramName, cell.Column.SqlDbType)
-                {
-                    Value = cell.Value
-                };
-                command.Parameters.Add(param);
-            }
-            string joinedInColumns = string.Join(", ", inColumns);
-            string joinedOutColumns = string.Join(", ", outColumns);
+            // Build the script and generate a command
+            ScriptBuildResult result = BuildInsertScript(true);
```

Comment:
Minor: it's nice to name the random bool parameter so I know what `true` actually indicates.